modules/unrulyBidAdapter.js

```diff
@@ -6,7 +6,7 @@ import { VIDEO } from '../src/mediaTypes'
 function configureUniversalTag (exchangeRenderer) {
   parent.window.unruly = parent.window.unruly || {};
   parent.window.unruly['native'] = parent.window.unruly['native'] || {};
-  parent.window.unruly['native'].siteId = parent.window.unruly['native'].siteId || exchangeRenderer.siteId;
+  parent.window.unruly['native'].siteId = parent.window.unruly['native'].siteId || exchangeRenderer.config.siteId;
```

Comment:
can you please add a check to verify `exchangeRenderer.config` is defined before trying to access `siteId`?

I notice this `exchangeRenderer` object comes from your serverBid, and you may guarantee that this is always defined, but we would prefer to be defensive here please !

Reply:
Want to void `Error: Cannot read property 'siteId' of undefined`

Reply:
Roger that—throwing meaningful exceptions unless `siteId` is available.

Reply:
@paprikka Thanks,

quick question, where exactly does this `exchangeRenderer.config` come from?

The Unruly server request?

Is it set by the publisher somewhere?

Reply:
`exchangeRenderer.config` comes from the `bidResponse`

Reply:
Right, okay.

So the suggestion here is to not throw an error, but instead use the log utils to log the error and return the expected value back to prebid core.

for example, Prebid Core expects the bids back as an array. So instead of throwing the error,

Can we update to log the error, and simply return back an empty array to Prebid core?

Thanks!

Reply:
Sure, done. Thanks!